plugins/standard_tests/newhelm/tests/safe.py

```diff
@@ -48,8 +49,9 @@ def get_metadata(self) -> TestMetadata:
 
     def get_dependencies(self) -> Mapping[str, ExternalData]:
         return {
-            "prompts_csv": WebData(
-                source_url=f"{self.data_source}/export?format=csv",
+            f"prompts_csv_{FOLDER_NAME}": GDriveData(
+                folder_url=self.folder_url,
+                filename=f"mls_ais_wg_safe_{self.hazard}_hazardous.csv",
```

Comment:
One high level concern I have is that before if someone renamed the csv or moved it into a different folder, the download would still work because it used the resource id which wouldn't change.

The improved clarity is probably worth it, but just wanted to raise the concern.

Reply:
Yeah, I see where you are coming from buy don't think it should be *too* much of a concern here.  I think we can safely make assumptions about things like the file names since these datasets are coming from the working group and we can establish an expected protocol. And also these folders are locked!